lib/node_modules/@stdlib/stats/base/range/test/test.ndarray.js

```diff
@@ -1,106 +1,114 @@
 /**
-* @license Apache-2.0
-*
-* Copyright (c) 2020 The Stdlib Authors.
-*
-* Licensed under the Apache License, Version 2.0 (the "License");
-* you may not use this file except in compliance with the License.
-* You may obtain a copy of the License at
-*
-*    http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
-*/
+ * @license Apache-2.0
```

Comment:
```suggestion
* @license Apache-2.0
```
@jalajk3004 Hey the license should not be tempered with here. Hence the CI error.

Reply:
Thanks @Neerajpathak07, Error is solved.
But I am still facing the problem with Repl
![image](https://github.com/user-attachments/assets/6d37f789-e3ab-4808-9d22-aa89d7bf00d6)

Reply:
well that's because the expected and the returned values do not match. Check your implementation once again and do go through the `repl.txt` file to see if everything is in-line.